server/controllers/finance/cash.js

```diff
@@ -281,3 +285,25 @@ function reference(req, res, next) {
   .catch(next)
   .done();
 }
+
+/**
+ * GET /cash/:project/:reference
+ * Check if the invoice is paid 
+ */
+function checkin(req, res, next) {
+  var project = req.params.project;
+  var reference = req.params.reference;
+
+  const sql =
+    `SELECT BUID(cash.uuid) AS uuid 
+      FROM cash
+    WHERE cash.project_id = ? AND cash.reference = ? AND
+    cash.uuid NOT IN (SELECT voucher.reference_uuid FROM voucher WHERE voucher.type_id = 10)`;
```

Comment:
Voucher 10 is used as being the entity_type value when it comes to a transaction reverse